recipes/sbpy/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "sbpy" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 07fe29c0c46af723043f78aa00914cffba1dceb01b90bc6f934810f982a8e76f
+  patches:
+    - setup.cfg.patch
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.13.0
+    - astropy >=3.0
+    - matplotlib
+    - ads
+    - synphot
+    - astroquery >=0.4.1.dev0
```

Comment:
Can this just be `>=0.4.1`?

Reply:
I think so, yes -- I've just updated the requirement